ui/cypress/integration/integration/settings_spec.js

```diff
@@ -0,0 +1,18 @@
+describe("Settings", () => {
+    describe("Service Meshes", () => {
+      before(() => {  
+        cy.selectProviderNone()
+
+        cy.visit("/settings");
+  
+        cy.get('[data-cy="tabServiceMeshes"]').click();
+      });
+      it("select, submit, and confirm", () => {
+        cy.get(".MuiInputBase-input").type("mesherylocal.layer5.io:10000{enter}");
+  
+        cy.get("[data-cy=btnSubmitMeshAdapter]").click();   
+
+        cy.get("[data-cy=adapterSuccessSnackbar]").should('exist')
+      });
+    });
+  });
```

Comment:
Format this file following the eslint configuration to be the same as the othert tests:

1. Use **two spaces** to ident
2. Delete blank spaces
3. Add a new line at the end of the file

Here is an image of the style errors:

![image](https://user-images.githubusercontent.com/2797052/110253248-e3be2c80-7f4e-11eb-859b-5f139e252eca.png)

If you're using VSCode you can use this extension: [vscode-trailingspaces](https://github.com/shardulm94/vscode-trailingspaces). If you're using another editor, there are other plugins/packages to help with this also.